cheddar/cheddar-metrics-intercom/src/test/java/com/clicktravel/cheddar/metrics/intercom/random/data/RandomDataGenerator.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2014 Click Travel Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.clicktravel.cheddar.metrics.intercom.random.data;
+
+import static com.clicktravel.common.random.Randoms.randomEmailAddress;
+import static com.clicktravel.common.random.Randoms.randomId;
+import static com.clicktravel.common.random.Randoms.randomIntInRange;
+import static com.clicktravel.common.random.Randoms.randomString;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.clicktravel.cheddar.metrics.MetricOrganisation;
+import com.clicktravel.cheddar.metrics.MetricUser;
+
+import io.intercom.api.Company;
+import io.intercom.api.CompanyCollection;
+import io.intercom.api.CustomAttribute;
+import io.intercom.api.User;
+
+@SuppressWarnings("rawtypes")
+public class RandomDataGenerator {
```

Comment:
Good idea to create this class but it is currently a mixture of Intercom-specific object generation and non-Intercom-specific (i.e. generic metric) object generation.

I think it can be split into two:

- RandomMetricDataGenerator: randomMetricOrganisation(), randomMetricUser(), randomMetricCustomAttributes()
- RandomIntercomDataGenerator: randomIntercomUser(), randomCompanyCollection(), randomCustomAttributes().